Backend/ServerModules/ServerBehavior.cs

```diff
@@ -11,8 +11,8 @@ public class ServerBehavior : WebSocketBehavior
         protected override void OnMessage(MessageEventArgs e)
         {
             string rawString= e.Data;
-            WebSocket socket = Context.WebSocket;
-            User? user = UserManager.GetUserBySocket(socket);
+            WebSocket currentSocket = Context.WebSocket;
```

Comment:
this can be named named socket again after wrapping the reflection send code in a SendToAll function

Reply:
Done